README.md

```diff
@@ -53,12 +53,21 @@ count-files -h
 count-files --help
 ```
 
-By default, the program counts or searches for files recursively in current working directory and all of its subdirectories. Any hidden files or folders will be ignored.
-The names of extensions are case insensitive by default. The results for `ini` and `INI` will be the same.
-
+By default, the program counts or searches for files recursively in current working directory and all of its subdirectories.  
+For fully supported operating systems (Linux, macOS, iOS, Windows), any hidden files or folders are ignored by default. For other operating systems in which Python can be run, this option to include/exclude hidden files is not available. And as a result, all existing files will be included.  
```

Comment:
Actually, I think that this is not the best default behaviour. Since most operating systems, apart from Windows, are unix-like and treat  files with names starting with dot (.gitignore) as hidden files, we should simply assume that at least these files are always hidden except on Windows.

Reply:
This was designed as an extension of CLI capabilities.
Identifying hidden folders and files is a task specific to the operating system. In other parts, the program does not depend on a specific OS.
The presence of BaseOS allows us not to assume, but to know the behavior of the program.
Earlier [versions](https://github.com/victordomingos/Count-files/blob/v.1.4/count_files/utils/file_handlers.py#L176) were intended for major operating systems. Currently, CLI can be run and tested on those operating systems where Python can run.
Some specific operating systems can simply be added to the list (in def get_current_os), as was done with Haiku.
We may also consider replacing the [sys.platform.startswith()](https://docs.python.org/3/library/sys.html#sys.platform) with the [platform.system()](https://docs.python.org/3/library/platform.html#platform.system) in [def get_current_os](https://github.com/victordomingos/Count-files/blob/master/count_files/platforms.py#L192).
`platform.system() == 'Linux' ` for more generalization.
This seems to cover most of these distributions: [examples](https://github.com/easybuilders/easybuild/wiki/OS_flavor_name_version)